.eslintrc.lang.js

```diff
@@ -0,0 +1,18 @@
+module.exports = {
+  env: {
+    browser: true,
+    es6: true,
+    node: true,
+    mocha: true
+  },
+  extends: [],
+  globals: {
+    Atomics: "readonly",
+    SharedArrayBuffer: "readonly"
+  },
+  parserOptions: {
+    ecmaVersion: 2015,
```

Comment:
AFAIK, that's the same as `es6: true` (`parserOptions` in current state can be dropped)

Reply:
Thanks.  I believe it is, but I also like it being explicit.